sample/src/androidTest/java/com/schibsted/spain/barista/sample/SwipeRefreshTest.java

```diff
@@ -0,0 +1,22 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.BaristaAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.BaristaSwipeRefreshActions.pullToRefresh;
+
+@RunWith(AndroidJUnit4.class) public class SwipeRefreshTest {
```

Comment:
This is probably an auto-formatting error. But for consistency let's put the class declaration below the annotation. 🙏